src/test/kotlin/com/google/androidstudiopoet/converters/ConfigPojoToAndroidModuleConfigConverterTest.kt

```diff
@@ -105,7 +106,7 @@ class ConfigPojoToAndroidModuleConfigConverterTest {
         val androidModuleConfig = converter.convert(configPOJO, 0, productFlavorConfigs, buildTypes)
         assertOn(androidModuleConfig) {
             hasLaunchActivity.assertTrue()
-            dependencies!!.assertEquals(listOf(DependencyConfig.ModuleDependencyConfig(ANDROID_MODULE_NAME_1)) + PURE_MODULE_DEPENDENCY_LIST)
+            dependencies!!.assertEquals(listOf(DependencyConfig.ModuleDependencyConfig(ANDROID_MODULE_NAME_1, DEFAULT_DEPENDENCY_METHOD)) + PURE_MODULE_DEPENDENCY_LIST)
```

Comment:
Can you please write a test that checks proper propagation of `method` from `FromToDependencyConfig` to `ModuleDependencyConfig`. Same for `ConfigPojoToModuleConfigConverter`

Reply:
`convert passes correct values to result AndroidModuleConfig` already does that for AndroidModule, just changed the value to not use DEFAULT_DEPENDENCY_METHOD

Reply:
Also modified `convert passes correct values to result ModuleConfig` to also test it on ConfigPojoToModuleConfigConverter